src/cdk-experimental/popover-edit/lens-directives.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ReplaySubject} from 'rxjs';
+import {Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {EDIT_PANE_SELECTOR} from './constants';
+import {closest} from './polyfill';
+import {EditRef} from './edit-ref';
+
+/**
+ * Options for what do to when the user clicks outside of an edit lens.
+ */
+export type PopoverEditClickOutBehavior = 'close' | 'submit' | 'nothing';
```

Comment:
I'd go with `'noop'` over `'nothing'` since we already use `noop` somewhere else